Doc/reference/expressions.rst

```diff
@@ -1443,8 +1443,9 @@ substring of *y*.  An equivalent test is ``y.find(x) != -1``.  Empty strings are
 always considered to be a substring of any other string, so ``"" in "abc"`` will
 return ``True``.
 
-For user-defined classes which define the :meth:`__contains__` method, ``x in
-y`` is true if and only if ``y.__contains__(x)`` is true.
+For user-defined classes which define the :meth:`__contains__` method, the
+``in`` operator coerces the result returned by :meth:`__contains__` to a boolean,
+Hence ``x in y`` is true if and only if ``y.__contains__(x)`` is true.
```

Comment:
Perhaps changing the example to `bool(y.__contains__(x))` would make the whole paragraph clearer?